modules-version.sh

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env bash
+
+# modules-version: A versatile tool for managing module versions in the Coder modules repository
+#
+# This script handles module versioning with support for module-specific tags in the format:
+# release/module-name/v1.0.0
+#
+# Features:
+# - Check that README.md versions match module tags (CI-friendly)
+# - Update module versions automatically or to specified versions
+# - Support for module-specific tags and versioning
+# - Create tags for new module versions
+#
+# Usage:
+#   ./modules-version.sh                                # Check all modules with changes
+#   ./modules-version.sh module-name                    # Check or update a specific module
+#   ./modules-version.sh --check                        # Check-only mode (no changes)
```

Comment:
Why is both `modules-version.sh` and `module-version.sh --check` needed, what's the difference?